Mathlib/Analysis/InnerProductSpace/l2Space.lean

```diff
@@ -91,9 +91,6 @@ open scoped BigOperators NNReal ENNReal Classical ComplexConjugate Topology
 
 noncomputable section
 
-local macro_rules | `($x ^ $y)   => `(HPow.hPow $x $y)
--- porting note: see lean4#2220
-
```

Comment:
I found out that this hack wasn't needed anywhere in the file, and it isn't needed to solve the universe error, so in the interest of minimising the diff with the mathport output I have removed this.